docs/CONTRIBUTING.md

```diff
@@ -606,6 +606,15 @@ manager process yourself. You can either get a project manager from one of the
 latest releases on [GitHub](https://github.com/enso-org/enso/releases), or build
 one using SBT `buildProjectManagerDistribution` command.
 
+Running development version of the IDE is possible via the `./run` script:
+
+```bash
+$ ./run start --dev --no-backend
```

Comment:
I would not advise people to use `--no-backend` in the docs until they know what they are doing. Maybe it is worth adding to the docs as an explanation?

Reply:
`--no-backend` is already mentioned in previous paragraph:
> IDE executable has `--no-backend`
flag that switches off the bundled backend. That requires you to run the project
manager process yourself. You can either get a project manager from one of the
latest releases on [GitHub](https://github.com/enso-org/enso/releases), or build
one using SBT `buildProjectManagerDistribution` command.

The new paragraph just explains how to achieve the same with manually built IDE.